app/views/publishers/new_features/reminder.html.slim

```diff
@@ -8,5 +8,9 @@
       = t(".page_title")
     p.govuk-body
       = t(".content")
-    = open_in_new_tab_link_to(t(".how_applications_work_link"), post_path(section: "get-help-hiring", subcategory: "how-to-create-job-listings-and-accept-applications", post_name: "accepting-job-applications-on-teaching-vacancies"), class: "govuk-body")
+    p.govuk-body
+      / govuk_link_to(t("app.opens_in_new_tab", link_text: text), href, target: "_blank", rel: "noreferrer noopener", **)
```

Comment:
:point_up: Is this a commented line left by accident? :scissors:

Reply:
Oops, yep sorry!